request.go

```diff
@@ -35,14 +40,14 @@ type RequestStatusProperties struct {
 
 //RequestError error of a request
 type RequestError struct {
-	StatusMessage string `json:"status"`
-	ErrorMessage  string `json:"message"`
-	StatusCode    int
+	Title       string `json:"title"`
```

Comment:
Plz, use `Message`.

Reply:
I think it is inconsistent with "title" JSON property if we name `Message`.

Reply:
Ok. then, ignore my comment.